xdc-plugin/tests/counter.xdc

```diff
@@ -0,0 +1,22 @@
+#set_property LOC R2 [get_ports led]
+set_property DRIVE I12 [get_ports {led[0]}]
```

Comment:
Vary the SLEW/IN_TERM/IOSTANDARD to have confidence that the code is not crossing wires

Reply:
Done